lib/assessment.js

```diff
@@ -8,30 +8,44 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
-};
+var min = reduce(function(previousValue, currentValue, index, array) {
+  return previousValue < currentValue ? previousValue : currentValue;
+)};
```

Comment:
So, here is the parts of this answer that would be considered good (as in full-points):

```js
var min = function(previousValue, currentValue, index, array) {
  return previousValue < currentValue ? previousValue : currentValue;
};
```

Remember, you were asked to write this function so it can be passed to `Array.reduce` by someone else. That means it's not your responsibility to be invoking the reduce operation in your answer.

There's also a syntax note: sets of brackets can nest, but cannot overlap. That is to say, this is fine: `{ ... ( ... [ ... ] ... ) ... }`, but this is not: `( ... { ... ) ... }`. It's not acceptable for the outer set of brackets to close before all sets of brackets opened within it are closed.

3/4 credit.